### ISSUE TITLE
[Accessibility]: Code Insights - Dashboard page announces a bunch of loading states

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#41801]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33493

### Problem description

Related to https://github.com/sourcegraph/sourcegraph/issues/41671

When you open the dashboard page you can hear a bunch of loading announcement that interrupt you flow when on the dashboard page and want to click on the crate insight/dashboard buttons.

### Expected behavior

Loading state should be announced only when you focus insight card and shouldn't be announced by default. 

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov

**Comment by vovakulikov** [comment]
Closed by https://github.com/sourcegraph/sourcegraph/pull/43970
